src/index.ts

```diff
@@ -42,11 +43,12 @@ export class GoogleToken {
   key: string|undefined;
   keyFile: string|undefined;
   iss: string|undefined;
-  sub: string;
+  sub: string|undefined;
   scope: string|undefined;
   rawToken: string|null;
   tokenExpires: number|null;
   email: string;
+  additionalClaims?: {};
```

Comment:
Why are all these fields public?

Reply:
It's kinda weird, but it's just how the library was originally written ::shrugs::